src/compressed_tensors/quantization/quant_args.py

```diff
@@ -115,10 +115,10 @@ def get_observer(self):
         """
         from compressed_tensors.quantization.observers.base import Observer
 
+        # No observer required for the dynamic case
         if self.dynamic:
-            # override defualt observer for dynamic, you never want minmax which
-            # keeps state across samples for dynamic
-            self.observer = "memoryless"
+            self.observer = None
```

Comment:
Can you add this validation logic in addition to this

```python3
Observer: Optional[str] = None

def model_validate_after(values):
    if values["observer"] is None:
        values["observer"] = "minmax"
    else:
        if self.dynamic:
            warning.warn("No observer is used for dynamic quantization, setting to None")
            values["observer"] = None
```

This also catches the case where the user tries to pass None for an observer in the non-dynamic case